docs/data/data-grid/group-pivot/BasicDetailPanels.js

```diff
@@ -73,7 +73,30 @@ function DetailPanelContent({ row: rowProp }) {
 }
 
 DetailPanelContent.propTypes = {
-  row: PropTypes.any.isRequired,
+  row: PropTypes.shape({
```

Comment:
Let's add `row` in the `shouldResolveObject` of `formattedTSDemos.js` (unless it removes interesting prop-types on other demos)